bc_obps/registration/models.py

```diff
@@ -140,6 +156,12 @@ class User(UserAndContactCommonInfo):
         blank=True,
         related_name="user_documents",
     )
+    app_role = models.ForeignKey(
+        AppRole,
+        on_delete=models.DO_NOTHING,
+        related_name="user_app_role",
```

Comment:
I suggest using plural names for related objects, as it retrieves multiple records from an `AppRole` instance rather than just one.